gleam_stdlib/test/gleam/float_test.gleam

```diff
@@ -137,3 +137,26 @@ pub fn truncate_test() {
   |> float.truncate
   |> expect.equal(_, -7)
 }
+
+pub fn max_test() {
+    float.max(0., 0.)
```

Comment:
Can you indent with 2 spaces here and in the int test file please? Thank you

Reply:
done!